webgpu.h

```diff
@@ -530,9 +539,9 @@ typedef struct WGPUBindGroupEntry {
 } WGPUBindGroupEntry;
 
 typedef struct WGPUBlendComponent {
-    WGPUBlendOperation operation;
     WGPUBlendFactor srcFactor;
     WGPUBlendFactor dstFactor;
+    WGPUBlendOperation operation;
```

Comment:
nit: I like marginally better before:

```
alphaBlend = {wgpu::BlendOperation::Add, wgpu::BlendFactor::One, wgpu::BlendFactor::DstAlpha}
// is more understandable than
alphaBlend = {wgpu::BlendFactor::One, wgpu::BlendFactor::DstAlpha, wgpu::BlendOperation::Add}

```

Reply:
I'd be happy to change this back, and we can reorder upstream too